.github/workflows/docs-deploy.yml

```diff
@@ -39,6 +39,22 @@ jobs:
         working-directory: ./website
         run: |
           make build
+      - id: cache-repos
+        name: cache-repos
+        uses: actions/cache@ab5e6d0c87105b4c9c2047343972218f562e4319 # v4.0.1
+        with:
+          path: website/build/repos
+          key: docs-cache-repo-build
+      - id: gpg-import
```

Comment:
@JanMa I'd comment out this section of the workflow and also in `genrepos.sh`, drop the line:

```
SignWith: E617DCD4065C2AFC0B2CF7A7BA8BC08C0F691F94
```

from the `conf/distributions` file and this should let you run it in your own fork (if you don't want to use `cipherboy.github.io/openbao`) without the GPG key.

Or, you could set up some secrets in your own repo and modify the signing subkey identifier to your own.